src/sentry/interfaces/exception.py

```diff
@@ -332,7 +332,7 @@ def get_api_meta(self, meta, is_public=False, platform=None):
         )
 
         stacktrace_meta = (
-            self.stacktrace.get_api_meta(meta, is_public=is_public, platform=platform)
+            self.stacktrace.get_api_meta(meta["stacktrace"], is_public=is_public, platform=platform)
```

Comment:
can we be sure that the `stacktrace` key exists?

Reply:
yeah, the line below was already checking for it: https://github.com/getsentry/sentry/pull/65852/files#diff-4c99ad258e77497dd3119edb46085864f9c40cd17fc325050a423ec989253902R336

Reply:
ah gotcha.